src/mpyl/reporting/targets/slack.py

```diff
@@ -107,13 +107,15 @@ def __init__(self,
 
     def send_report(self, results: RunResult, text: Optional[str] = None) -> ReportOutcome:
         try:
-            user_info = self.__get_user_info(results.run_properties.details.user_email)
+            email = results.run_properties.details.user_email
+            user_info = self.__get_user_info(email)
 
             if not self._channel and user_info.initiator:
                 self._channel = self.__open_conversation_with_user(user_info.initiator)
 
             if not self._channel:
-                raise ValueError('Channel not explicitly set and initiator could not be determined')
+                return SlackOutcome(success=False, exception=ValueError(
```

Comment:
Seems the build fails on this, which means pr's that are build by you cannot be merged. It should warn but not fail the build, right?